pkg/event/events.go

```diff
@@ -59,11 +59,11 @@ func NewPolicyAppliedEvent(source Source, engineResponse engineapi.EngineRespons
 		res = fmt.Sprintf("%s %s", resource.GetKind(), resource.GetName())
 	}
 
-	hasValidate := engineResponse.Policy().GetSpec().HasValidate()
-	if hasValidate {
-		fmt.Fprintf(&bldr, "%s: pass", res)
-	} else {
+	hasMutate := engineResponse.Policy().GetSpec().HasMutate()
+	if hasMutate {
 		fmt.Fprintf(&bldr, "%s is successfully mutated", res)
+	} else {
+		fmt.Fprintf(&bldr, "%s: pass", res)
```

Comment:
Why changing this block?

Reply:
In case of verifyImages, it was producing an event having "resource is successfully mutated" because I was only checking whether the rule is validate or not.

So I changed the condition to check if the rule is a mutate rule. So any rules rather than mutation should have a msg "resource pass" except for the mutation, it will have "resource is successfully mutated"

Reply:
What about generate and cleanup rules?

Reply:
for generate/mutate existing rules, events messages are handled in `NewBackgroundSuccessEvent()`
for cleanup rules, There are no events generated.

Reply:
In addition, I wonder if we can create new methos for the cleanup policies as well. The goal here is to have a single source of all kyverno generated events.
For example we can create `NewCleanupPolicyEvent` and use it in the cleanup controller. 
I amn't sure if there's a reason behind generating events in the cleanup handler itself.

Reply:
>In addition, I wonder if we can create new methos for the cleanup policies as well. The goal here is to have a single source of all kyverno generated events.

Agreed, we can change the cleanup event generation in a separate PR.

Reply:
>It's like a generic events for all policies. What's your thoughts on that ?

I would check the specific rule types to build the event message, this should be a good start. i.e., if it's a mutate rule st the message to "is successfully mutated", if it's a validate or imageVerify rule, set the message to pass.